src/components/Layouts/components/Notification/ItemNotification.tsx

```diff
@@ -0,0 +1,44 @@
+import classNames from 'classnames/bind'
+import styles from './Notification.module.scss'
+import { NotificationInterface } from '~/interfaces/notification'
+import moment from 'moment'
+
+const cx = classNames.bind(styles)
+
+interface ItemNotificationInterface {
+  data: NotificationInterface
+}
+const ItemNotification = ({ data }: ItemNotificationInterface) => {
+  return (
+    <div className={cx('notification', { no_seen: !data?.seen })}>
```

Comment:
Does call cx() func everytime in the component affect to the performance? If component has effect trigger other render, the cx() will be call again